rm/src/main/java/io/seata/rm/AbstractRMHandler.java

```diff
@@ -153,7 +152,7 @@ public AbstractResultMessage onRequest(AbstractMessage request, RpcContext conte
 
     @Override
     public void onResponse(AbstractResultMessage response, RpcContext context) {
-        throw new ShouldNeverHappenException();
+        LOGGER.warn("client received response msg [{}] from tc server, bug without any processing logic.", response.getMsg());
```

Comment:
if it's normal，response.getMsg() is null.

Reply:
ok, done